crates/data_components/Cargo.toml

```diff
@@ -49,6 +49,10 @@ odbc-api = { workspace = true, optional = true }
 postgres-native-tls = { version = "0.5.0", optional = true }
 r2d2 = { workspace = true, optional = true }
 rdkafka = { version = "0.36.2", optional = true }
+
+[target.'cfg(windows)'.dependencies]
```

Comment:
Should this still be optional?
rdkafka = { version = "0.36.2", features = ["cmake-build"], optional=true }

Reply:
Yes